.github/workflows/ci.yml

```diff
@@ -20,7 +23,7 @@ jobs:
       - name: Check if Docker image exists
         id: check
         run: |
-          if docker manifest inspect cometbft/cometbft-db-testing:v1.0.3 &>/dev/null; then
+          if docker manifest inspect cometbft/cometbft-db-testing:v1.0.4 &>/dev/null; then
```

Comment:
I don't like this, but I have not found a way around it. We want to a) use an existing image until the Dockerfile changes and b) continually build from scratch (including subsequent PRs) from that point on.